app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] Please the group's page should match the design, the nav color and the button color should be green.
- [ ] The background color should match the design the distance between cards should match the design.
- [ ] The button should match the design in its width and color ,the inner text should be in capital letters
![image](https://user-images.githubusercontent.com/74744787/219894461-3a5a5336-07f5-4c14-a5b9-edf9570a0e22.png)
![image](https://user-images.githubusercontent.com/74744787/219894688-c6693572-3b86-405c-8fe5-042c3787e88d.png)